packages/layer-tests/src/e2e/runner.rs

```diff
@@ -388,6 +392,39 @@ async fn run_test(
             }
             Trigger::BlockInterval { .. } => vec![TriggerId::new(1337)],
             Trigger::Cron { .. } => vec![TriggerId::new(1338)],
+            Trigger::AtProtoEvent { .. } => {
+                let sequence: u64 = 1339;
+                let trigger_id = TriggerId::new(sequence);
+
+                let record_payload = input_bytes.clone().unwrap_or_default();
+                let record_text = String::from_utf8_lossy(&record_payload).to_string();
+
+                let atproto_data = TriggerData::AtProtoEvent {
+                    sequence: sequence as i64,
+                    timestamp: 0,
+                    repo: "did:example:alice".to_string(),
+                    collection: "app.bsky.feed.post".to_string(),
+                    rkey: "rkey-1".to_string(),
+                    action: AtProtoAction::Create,
+                    cid: Some("bafytestcid".to_string()),
+                    record: Some(json!({ "text": record_text })),
+                    rev: Some("rev-test".to_string()),
+                    op_index: Some(0),
+                };
+
+                let req = SimulatedTriggerRequest {
+                    service_id: service_deployment.service.id(),
+                    workflow_id: first_workflow_id.clone(),
+                    trigger: trigger.clone(),
+                    data: atproto_data,
+                    count: 1,
+                    wait_for_completion: true,
+                };
+
+                clients.http_client.simulate_trigger(req).await?;
```

Comment:
Noted at high-level, this is good enough to unblock the PR from getting in, but misses testing things that can go wrong real-world.

https://github.com/Lay3rLabs/WAVS/issues/1101 should be done in the near future, before we advertise using this trigger too much